csp_collector.go

```diff
@@ -77,6 +77,9 @@ func validateViolation(r CSPReport) error {
 		"chromenull://",
 		"resource://",
 		"about:blank",
+		"android-webview",
+		"ms-browser-extension",
+		"http://adblockers.opera-mini.net",
```

Comment:
For android-webview and ms-browser-extension, I had them in my CSP reports.

For adblockers.opera-mini.net, I had it at the same time with opera:// (opera://js-inject, see https://github.com/nico3333fr/CSP-useful/tree/master/csp-wtf#opera-mini-double-wtf and other issues), these two came at the same time.

Anyway, I do agree, it is a bit hard to find documentation for these issues :-\